src/Controller/Api/PersonController.php

```diff
@@ -494,6 +499,26 @@ public function getPerson(Person $person): Response
             'country' => $person->getCountry(),
             'organization' => $person->getOrganization(),
             'position' => $person->getPosition()
-        ]);
+        );
+
+        $account = $person->getAccount();
+        if ($account instanceof Account) {
+            $posix = $account->isPosix();
+            $posixUsername = $account->getUsername();
+        }
+
+        $isMe = (($currentAccount instanceof Account) && $person->getId() === $currentAccount->getPerson()->getId()) ? true : false;
```

Comment:
```suggestion
                $isMe = ($currentAccount instanceof Account) && $person->getId() === $currentAccount->getPerson()->getId();

```

Reply:
ended up adding () for clarity, but cleaned up redundancy